### DISCUSSION TITLE
trigger on workflow completion

### DISCUSSION THREAD

**Discussion by staskh** [aws/amazon-genomics-cli#488] [Q&A]
Hi,

How can I trigger lambda  or other kind of event on workflow successful completion ( or failure)? Any examples will be very helpful.

Regards
Stas

**Comment by markjschreiber** [comment]
This cannot be done using the AGC CLI, however you can monitor state transition events in AWS Batch using AWS EventBridge. 

There is an example here https://docs.aws.amazon.com/batch/latest/userguide/batch_cwet.html
